# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../zsnes/zsnes-1.51-gcc44.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./tools/strutil.h.orig	2011-04-13 14:45:58.540078111 +0200
+++ ./tools/strutil.h	2011-04-13 14:46:08.496078199 +0200
@@ -23,6 +23,7 @@
 #define STRUTIL_H
 
 #include <string>
+#include <cstring>
 #include <vector>
 #include <cctype>
 
--- ./tools/depbuild.cpp.orig	2011-04-13 14:48:38.024077820 +0200
+++ ./tools/depbuild.cpp	2011-04-13 14:53:58.388077827 +0200
@@ -24,6 +24,7 @@
 #include <iostream>
 #include <string>
 #include <cstdio>
+#include <cstdlib>
 using namespace std;
 
 #include "fileutil.h"
--- ./parsegen.cpp.orig	2011-04-13 14:51:18.012083415 +0200
+++ ./parsegen.cpp	2011-04-13 14:51:40.708079442 +0200
@@ -26,6 +26,7 @@
 #include <iostream>
 #include <fstream>
 #include <string>
+#include <cstring>
 #include <vector>
 #include <algorithm>
 #include <sstream>
